daemon/cmd/kube_proxy_replacement_test.go

```diff
@@ -11,9 +11,7 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 )
 
-type KPRSuite struct {
-	orig kprConfig
```

Comment:
Given the naming, I presume this was supposed to store & restore the KPR config so that global state doesn't impact later test runs? Should we just implement the restore piece rather than dropping this?

Reply:
This was a left over from earlier refactoring which separated this out from daemon_test.go. This test now calls `option.Config.Populate` and does not rely on it being in some reliable state. Other tests need to do the same as well, so we can drop the restoring.